include/h2o/socket/evloop.h

```diff
@@ -88,9 +88,14 @@ static inline uint64_t h2o_now_nanosec(h2o_evloop_t *loop)
     return loop->_now_nanosec;
 }
 
-static inline uint64_t h2o_evloop_get_execution_time(h2o_evloop_t *loop)
+static inline uint64_t h2o_evloop_get_execution_time_millisec(h2o_evloop_t *loop)
 {
-    return loop->exec_time_counter.average;
+    return loop->exec_time_nanosec_counter.average * 1000000;
```

Comment:
Is the conversion correct? I think it needs to be `/ 1000000` considering that the LHS says "nanosec" and the function name says "millisec." (Yes, having the unit expressed in the names helps a lot!)

Reply:
D'oh! You're right, it's incorrect. Embarrassing 😅 Addressed in: 56e54560